src/pages/tutorials/index.tsx

```diff
@@ -42,44 +42,29 @@ function TutorialItem({ item }: { item: Project }) {
   const { githubId } = item
   return (
     <Link to={`/tutorial/${githubId}`}>
-      <article className="rounded-2xl overflow-hidden cursor-pointer border group z-[1] duration-200 ease-in hover:border hover:border-opacity-80 hover:bg-white/10 relative w-full transition-all hover:scale-[0.998]">
+      <article className="relative z-[1] hover:bg-white/10 border hover:border hover:border-opacity-80 rounded-2xl w-full transition-all duration-200 cursor-pointer overflow-hidden ease-in group hover:scale-[0.998]">
         <div className="relative flex flex-col">
-          <div className="absolute flex items-center justify-center p-1 rounded-full top-2 left-2 bg-muted">
+          <div className="top-2 left-2 absolute flex justify-center items-center bg-muted p-1 rounded-full">
             <LucideBookText className="w-4 h-4" />
           </div>
           {/* box */}
           <div className="h-48 overflow-hidden">
-            <img src={item.owner.avatarUrl} alt={item.owner.login} className="object-cover w-full h-full" />
+            <img src={item.owner.avatarUrl} alt={item.owner.login} className="w-full h-full object-cover" />
           </div>
-          <div className="p-4 overflow-hidden lg:p-6 ">
+          <div className="p-4 lg:p-6 h-60">
             {/* name */}
-            <div className="flex items-center w-full gap-2">
-              <div>
-                <Button
-                  variant="link"
-                  className="flex-1 px-0 overflow-hidden text-xl font-bold whitespace-nowrap text-ellipsis"
-                >
-                  <span
-                    className="z-10"
-                    onClick={(e) => {
-                      e.preventDefault()
-                      e.stopPropagation()
-                      window.open(item.htmlUrl, '_blank')
-                    }}
-                  >
-                    {item.name}
-                  </span>
-                </Button>
-              </div>
-            </div>
+            <Button asChild variant="link" className="px-0 !line-clamp-2 h-16 font-bold text-xl break-all">
```

Comment:
Keep the original code please. It is used to jump to github.
And <a> included in <a> will cause warnings.

Reply:
"Alright, it was my mistake. I will make the necessary changes."